packages/console/src/components/ui/console.module.css

```
@@ -113,14 +113,14 @@
 }
 
 .listItem[data-active='true'] {
-  border-left-color: var(--app-service-color);
-  background: var(--app-service-bg);
+  border-left-color: var(--app-violet);
+  background: var(--app-surface-violet);
 }
 
 .listItemPadded {
   composes: listItem;
```

Stylelint (17.14.0)

[error] Expected "listItem" to be "listitem" (value-keyword-case)

(value-keyword-case)